experiment/graphql_issue_example.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# USAGE: find_issues.py <github_token>
+from __future__ import print_function
+
+import sys
+import json
+
+import requests
+
+GITHUB_API_URL = "https://api.github.com"
+
+
+def do_github_graphql_request(query, token, variables=None):
+    """performs a requests.put with the correct headers for a GitHub request.
+
+    see: https://developer.github.com/v4/
+
+    Args:
+    query: the graphQL query string (excluding the variables section)
+    token: a GitHub API token string
+    variables: a dict of key=value variables that will be sent with the query
+
+    Returns:
+    A requests.Response object
+    """
+    url = "https://api.github.com/graphql"
+    headers = {
+        "Authorization": "bearer " + token,
+    }
+    data = json.dumps({"query": query, "variables": json.dumps(variables)})
+    return requests.post(url, headers=headers, data=data)
+
+
+def get_issues(owner, name, token, after=None):
+    """returns the result of do_github_graphql_request for a repo issues query.
+
+    This query requests the first 100 issues for a repo with the first 100
+    assignee logins and labels as well as the issue title, number, state,
+    creation time and the pageInfo for getting the next page of results
+
+    Args:
+    owner: the GitHub repo owner as in github.com/kubernetes/test-infra ->
+            owner="kubernetes"
+    name: this GitHub repo name as in github.com.kubernetes/test-infra ->
+            name = "test-infra"
+    token: a GitHub API token string
+    after: this should be None or the endCursor from pageInfo
+
+    Returns:
+    A requests.Response object
+    """
+    query = ("query($owner:String!, $name:String!, $after:String) {\n"
```

Comment:
Can you change this to a """ string for clarity here?

Reply:
done.